docs/content/configuration/storage.md

```diff
@@ -114,110 +115,151 @@ After this you can pass you credentials inside `config.toml` file
             Key = "YOUR_MINIO_SECRET_KEY"
-    
+
             # Secret Key for Minio storage
             # Env override: ATHENS_MINIO_SECRET_ACCESS_KEY
             Secret = "YOUR_MINIO_SECRET_KEY"
-    
+
             # Timeout for networks calls made to Minio in seconds
             # Defaults to Global Timeout
             Timeout = 300
-    
+
             # Enable SSL for Minio connections
             # Defaults to true
             # Env override: ATHENS_MINIO_USE_SSL
             EnableSSL = false
-    
+
             # Minio Bucket to use for storage
             # Env override: ATHENS_MINIO_BUCKET_NAME
             Bucket = "gomods"
-            
+
 ## DigitalOcean Spaces
 
 For Athens to communicate with [DigitalOcean Spaces](https://www.digitalocean.com/products/spaces/), we are using Minio driver because DO Spaces tries to be [fully compatible with it](https://developers.digitalocean.com/documentation/spaces/).
-Also configuration for this storage looks almost the same in our proxy as for [Minio](#minio). 
+Also configuration for this storage looks almost the same in our proxy as for [Minio](#minio).
 
 ##### Configuration:
 
     # StorageType sets the type of storage backend the proxy will use.
     # Env override: ATHENS_STORAGE_TYPE
     StorageType = "minio"
-    
-    [Storage] 
+
+    [Storage]
         [Storage.Minio]
             # Address of DO Spaces storage
             # Env override: ATHENS_MINIO_ENDPOINT
             Endpoint = "YOUR_ADDRESS.digitaloceanspaces.com"
-    
+
             # Access Key for Minio storage
             # Env override: ATHENS_MINIO_ACCESS_KEY_ID
             Key = "YOUR_DO_SPACE_KEY_ID"
-    
+
             # Secret Key for DO Spaces storage
             # Env override: ATHENS_MINIO_SECRET_ACCESS_KEY
             Secret = "YOUR_DO_SPACE_SECRET_KEY"
-    
+
             # Timeout for networks calls made to DO Spaces storage in seconds
             # Defaults to Global Timeout
             Timeout = 300
-    
-            # Enable SSL 
+
+            # Enable SSL
             # Env override: ATHENS_MINIO_USE_SSL
             EnableSSL = true
-    
+
             # Space name in your DO Spaces storage
             # Env override: ATHENS_MINIO_BUCKET_NAME
             Bucket = "YOUR_DO_SPACE_NAME"
-            
+
             # Region for Minio storage
             # Env override: ATHENS_MINIO_REGION
             Region = "YOUR_DO_SPACE_REGION"
-            
\ No newline at end of file
+
+## Alibaba OSS
+
+For Athens to communicate with [Alibaba Cloud Object Storage Service](https://www.alibabacloud.com/product/oss), we are using Minio driver.
+Also configuration for this storage looks almost the same in our proxy as for [Minio](#minio).
+
+##### Configuration:
+
+    # StorageType sets the type of storage backend the proxy will use.
+    # Env override: ATHENS_STORAGE_TYPE
+    StorageType = "minio"
+
+    [Storage]
+        [Storage.Minio]
+            # Address of DO Spaces storage
```

Comment:
Could we change that to  something like `# Address of Alibaba OSS`? And also all occurences of `Minio/DO Spaces` to `Alibaba OSS` in other parameters descriptions :slightly_smiling_face:

Reply:
Oops :disappointed_relieved: Good spotting!